src/components/ImageBlock.tsx

```diff
@@ -17,14 +17,16 @@ const ImageBlock = (props: Props): JSX.Element => {
   const baseClass = isFocused ? '_legoEditor_imgSelected' : '_legoEditor_img';
   return (
     <div {...attributes}>
-      <div contentEditable={false}>
-        <img
-          src={src}
-          alt="Failed to load image..."
-          className={cx(baseClass, uploading && '_legoEditor_imgUploading')}
-        />
-      </div>
-      {children}
+      <>
```

Comment:
Why? This component was only returning one div, what good does fragments here? Or am I blind?

Reply:
There was a type error (likely after the react update) that the children prop in a div could only be an array of components or a component. This has always worked but I guess the new typing simplifies stuff a bit 🤷

Reply:
> children prop in a div could only be an array of components or a component

hmm, ait 🤷🏻 I would think that with `children` being 0, 1 or N components the previous syntax should be legal. Since there is already a div around the whole thing. Which gives:
```jsx
// In the case of no children
<div>
  <div content></div content>
</div>

// In the case of 1 child
<div>
  <div content></div content>
  <h1> </h1>
</div>

// In the case of N children
<div>
  <div content></div content>
  <h1> </h1>
  <h1> </h1>
</div>
```

But no matter, I'll allow it

Reply:
I think in the previous case `children` on the div would be `[div, [children]]`. And fore some reason that's okay in fragments but not other stuff 🤷

Reply:
The `children` prop here is typed as `unknown`, would it work without the fragment if it was `ReactNode`?

Reply:
aaah, yeah that's the reason lol. Thanks :1st_place_medal:

Reply:
Nice!!! ty @eikhr